roboflow/__init__.py

```diff
@@ -118,6 +120,8 @@ def login(workspace=None, force=False):
 
     else:
         raise RuntimeError("Error logging in")
```

Comment:
This is not very description to user nor providing information if he opens a PR with the error, just call

```
r_login.raise_for_status()
```

and remove the condition